x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/views/analytics.tsx

```diff
@@ -140,3 +157,11 @@ export const Analytics: React.FC = () => {
     </AnalyticsLayout>
   );
 };
+
+export const ViewAllButton: React.FC<{ to: string }> = ({ to }) => (
```

Comment:
Is it okay to write props directly inside the `React.FC<>`?

Reply:
I mean, it's okay in that Typescript isn't throwing an error on it and it's validating correctly :) It's not common in our current codebase but IMO this is essentially the same as doing this:

```ts
const someFn = ({ someObjValue }: { someObjValue: string }) => {}
```

as opposed to
```ts
interface SomeFn {
  someObjValue: string
}
const someFn = ({ someObjValue }: SomeFn) => {}
```

They're both valid Typescript, and depending on the complexity of the incoming params it's nicer to do an inline interface over pulling it out.